internal/performance/helper.go

```diff
@@ -0,0 +1,101 @@
+// +build performance
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package performance
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	// Warning: package github.com/golang/protobuf/proto is deprecated: Use the
+	// "google.golang.org/protobuf/proto" package instead.  (SA1019).
+	// Use deprecated package as mako isn't compatible with
+	// github.com/golang/protobuf/proto yet
+	"github.com/golang/protobuf/proto"
+	"github.com/google/mako/go/quickstore"
+
+	qpb "github.com/google/mako/proto/quickstore/quickstore_go_proto"
+	mpb "github.com/google/mako/spec/proto/mako_go_proto"
+)
+
+const (
+	benchmarkConfigFile = "export_test_benchmark.config"
+)
+
+// setup sets up client used for performance test
+func setup(tb testing.TB) (*quickstore.Quickstore, func(context.Context)) {
+	var (
+		microservice string
+	)
+
+	benchmarkConfig := &mpb.BenchmarkInfo{}
+
+	{
+		p := os.Getenv("MAKO_PORT")
```

Comment:
env var configuration should use a config struct
https://github.com/google/exposure-notifications-server/blob/main/internal/publish/config.go#L39

and then we use a library to parse the environment

Reply:
sg, done